src/index.html

```diff
@@ -7,7 +7,73 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars stars--0">
+      <a href="#" class="stars__star"></a>
+
+      <a href="#" class="stars__star"></a>
+
+      <a href="#" class="stars__star"></a>
+
+      <a href="#" class="stars__star"></a>
+
+      <a href="#" class="stars__star"></a>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
    </div>
```
you need to add spaces between blocks of code, here it's not necessary, also use div, there is no need to make stars as links